package.json

```
@@ -150,6 +150,12 @@
           ],
           "description": "Arguments to pass to 'zig' for running tests. Supported variables: ${filter}, ${path}."
         },
+        "zig.zls.debugAdapter": {
```

This is not a ZLS option.
```suggestion
        "zig.debugAdapter": {
```